pkg/migrations/backfill.go

```diff
@@ -151,7 +151,11 @@ func (b *batcher) updateBatch(ctx context.Context, conn db.DB) error {
 
 		// Execute the query to update the next batch of rows and update the last PK
 		// value for the next batch
-		err := tx.QueryRowContext(ctx, query).Scan(&b.lastValues)
+		wrapper := make([]interface{}, len(b.lastValues))
```

Comment:
nit: interace{} = any

```suggestion
		wrapper := make([]any, len(b.lastValues))
```